### PR TITLE
Fix beam unit tests

### PR DESCRIPTION
**Pull request by asattely**
These are the unit test adjustments for PR https://github.com/musescore/MuseScore/pull/10095

Beam slope has been changed, so the unit tests needed to be changed as well.

### CODE REVIEW

**Comment by asattely** [comment]
ignore this until it builds, i'm going to have to incorporate some changes from the draft PR before this will pass checks

**Review by oktophonie — APPROVED** [review]

**Comment by RomanPudashkin** [comment]
You should fix the commit messages. For example, the last commit should describe what kind of bug it fixes. "extra tweaks" should probably be combined with the first commit. The PR description (and its title) should be corrected as well

Screenshot 2022-02-23 at 11 33 05

File: src/engraving/libmscore/beam.cpp [hunk 1]
```diff
@@ -617,28 +617,30 @@ int Beam::computeDesiredSlant(int startNote, int endNote, int middleLine, int di
     }
     // for 2-indexed interval i (seconds, thirds, etc.)
     // maxSlopes[i] = max slope of beam for notes with interval i
-    static const int maxSlopes[] = { 1, 2, 3, 4, 5, 6 };
-
+    static const int maxSlopes[] = { 1, 2, 3, 4, 5, 6, 7 };
```
Comment (RomanPudashkin) [inline src/engraving/libmscore/beam.cpp:620]: Better to use constexptr std::array here

File: src/engraving/libmscore/beam.cpp [hunk 2]
```diff
@@ -617,28 +617,30 @@ int Beam::computeDesiredSlant(int startNote, int endNote, int middleLine, int di
...
+    static const int numSlopes = sizeof(maxSlopes) / sizeof(maxSlopes[0]);
     // calculate max slope based on distance between first and last chords
     qreal beamWidth = _elements[_elements.size() - 1]->stemPos().x() - _elements[0]->stemPos().x();
     beamWidth /= spatium();
-    int maxSlope = 6;
+    int maxSlope;
```
Comment (RomanPudashkin) [inline src/engraving/libmscore/beam.cpp:625]: This should be initialized with some value by default

**Comment by asattely** [comment]
Squashed, made improvements, as long as all the tests pass this one should be good to go

**Review by oktophonie — APPROVED** [review]

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**